src/workerd/api/worker-rpc.c++

```diff
@@ -1361,7 +1363,7 @@ private:
         "arguments, the server must use class-based syntax (extending WorkerEntrypoint) "
         "instead.");
 
-    KJ_STACK_ARRAY(v8::Local<v8::Value>, arguments, kj::max(argCountFromClient + 2, arity), 8, 8);
+    v8::LocalVector<v8::Value> arguments(js.v8Isolate, kj::max(argCountFromClient + 2, arity));
```

Comment:
Should we just change `KJ_STACK_ARRAY` implementation?

Reply:
KJ_STACK_ARRAY is a kj thing that is not specific to v8 at all. It wouldn't make sense to change it's implementation here.